realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -1492,6 +1493,11 @@ public Long call() throws Exception {
 
                         return handoverTableViewPointer;
 
+                    } catch (BadVersionException e) {
+                        // In some rare race conditions, this can happen. In that case, just ignore the error.
+                        RealmLog.d("findAllAsync handover could not complete due to a BadVersionException. " +
```

Comment:
This is only being logged in Debug mode, which means that normal users do not see this message. This message is mostly for internal debugging (or we can ask users to enable it).